packages/toolkit/src/view/model/view-model/ModelOverview.tsx

```diff
@@ -47,7 +48,11 @@ const selector = (store: InstillStore) => ({
 });
 
 const convertTaskNameToPayloadPropName = (taskName?: ModelTask) =>
-  taskName ? taskName.replace("TASK_", "").toLowerCase() : null;
+  taskName
+    ? convertSentenceToCamelCase(
+        taskName.replace(/TASK_|_/g, (d) => (d === "TASK_" ? "" : " "))
+      )
```

Comment:
@orangecoloured Please leave comment for this kind of code, especially address what is your intention

Reply:
Done!